google-api-go-generator/gen.go

```diff
@@ -714,11 +727,13 @@ func (a *API) GenerateCode() ([]byte, error) {
 	pn("var _ = strings.Replace")
 	pn("var _ = context.Canceled")
 	pn("var _ = internaloption.WithDefaultEndpoint")
+	pn("var _ = internaloption.WithDefaultMTLSEndpoint")
```

Comment:
No need for this, only one symbol. is needed per package we import.

```suggestion
```

Reply:
done